orca-core/src/test/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfoSpec.groovy

```diff
@@ -55,11 +57,48 @@ class PackageInfoSpec extends Specification {
       [["fileName": "test-package_1.0.0.deb"]]    | "test-package"                                 | "test-package_1.0.0"
       [["fileName": "test-package_1.0.0.deb"]]    | "another-package"                              | "another-package"
       [["fileName": "test-package_1.0.0.deb"]]    | "another-package test-package"                 | "another-package test-package_1.0.0"
-
+      [["fileName": "test-package_1.0.0.deb"]]    | "ssh://git@stash.corp.netflix.com:7999/uiplatform/nodequark.git?v1.0.51"| "ssh://git@stash.corp.netflix.com:7999/uiplatform/nodequark.git?v1.0.51"
```

Comment:
if it is a url we just return from the createAugmentedRequest method

`    if (isUrl(request.package)) {
      return request
    }`